File: docs/SDK-Documentation/getting-started/create-new-context-request/python/withPrefetchedData.py
# define a new context request
    context_config = ContextConfig()
    context_config.publish_delay = 10
    context_config.refresh_interval = 5
    context_config.units = {"session_id": "bf06d8cb5d8137290c4abb64155584fbdb64d8",
                            "user_id": "12345"}
    
    context_config = ContextConfig()
    ctx = sdk.create_context(context_config)
    ctx.wait_until_ready_async()

File: docs/APIs-and-SDKs/SDK-Documentation/getting-started/create-new-context-request/js/withPreFetchedData.js
// In your server-side code
const serverSideParams = {
  units: {
    user_id: '123',
    anonymous_id: '456',
    session_id: '789',
    // ...
  },
};

const serverSideContext = sdk.createContext(serverSideParams);

await serverSideContext.ready();

const serverSideData = serverSideContext.data();

// In your client-side code
const clientSideOptions = {
  publishDelay: 100, // ms
};

const clientSideContext = sdk.createContextWith(serverSideParams, serverSideData, clientSideOptions);
